docs/apache-airflow/concepts/cluster-policies.rst

```diff
@@ -57,12 +57,17 @@ This policy checks if each DAG has at least one tag defined:
 Task policies
 -------------
 
-Here's an example of enforcing a maximum timeout policy on every task:
+Here's an example of enforcing a maximum timeout policy on every task::
 
-.. literalinclude:: /../../tests/cluster_policies/__init__.py
```

Comment:
Why not using literalinclude from cluster_policies?

Reply:
@potiuk when using literalinclude, it raises `redefinition` error.

<img width="686" alt="image" src="https://user-images.githubusercontent.com/8676247/167551873-1c26791f-3246-4a84-b6bb-8c1d78124ffd.png">

Can I create new file `tests/cluster_policies/example_task_policy.py` for adding that code?

Reply:
Just rename the other task_policy

Reply:
Could you explain it more?
If I rename `task_policy` method (eg. `two_days_task_policy`), then the code in the document and the contents of the document do not match. I think it is weird.
![image](https://user-images.githubusercontent.com/8676247/167618723-66fa8791-1ed7-4159-aab7-9f088d57fc96.png)

Reply:
Why don't you replace it cluster_policy() with "example_task_policy" for example? 

There is no reason why task_policy should be named  always "task_policy". This is why it was named "cluster_policy" (historically we only had cluster policy). But this is quite ok to have a  method named differently and import it as different method name in python.  There is a "import x as y" and patching also work this way.

Reply:
I was just thinking of a way to declare a method only in `airflow_local_settings.py` file...

I will change method name `cluster_policy` to `example_task_policy`!